src/ember-app.js

```diff
@@ -60,17 +65,27 @@ class EmberApp {
    */
   buildSandbox(distPath, sandboxClass, sandboxGlobals) {
     let sandboxRequire = this.buildWhitelistedRequire(this.moduleWhitelist, distPath);
-    let config = this.appConfig;
-    function appConfig() {
-      return { default: config };
+    const config = this.config;
+    const appName = this.appName;
+    function fastbootConfig(key) {
+      if (!key && config) {
```

Comment:
I don't think this guard is needed, defaulting the `key` here when not provided to `appname/config/environment` (regardless if `config` is available) seems totally fine, no?

Reply:
Yeah I might have forgotten to clean this up..